lib/src/shared/status.dart

```diff
@@ -241,5 +279,55 @@ class GrpcError implements Exception {
   int get hashCode => code.hashCode ^ (message?.hashCode ?? 17);
 
   @override
-  String toString() => 'gRPC Error ($code, $message)';
+  String toString() =>
+      'gRPC Error (code: $code, codeName: $codeName, message: $message, details: $details)';
+}
+
+/// Given a status code, return the name
+String getStatusCodeValue(int code) {
+  if (code > Code.values.length - 1) {
+    return Code.UNKNOWN.name;
+  } else {
+    return Code.values
+        .firstWhere((e) => e.value == code, orElse: () => Code.UNKNOWN)
+        .name;
+  }
+}
+
+/// Parse error details into the right kind of GeneratedMessage.
+GeneratedMessage parseGeneratedMessage(Any any) {
```

Comment:
Consider giving this function a more descriptive name, e.g. `parseErrorDetailsFromAny(...)`

Reply:
Fixed in https://github.com/bottlepay/grpc-dart/commit/cd197160a01b0b1ac641eccd433d5b4f55f20f9c